dataprofiler/profilers/profile_builder.py

```diff
@@ -80,15 +81,32 @@ def __init__(self, df_series, sample_size=None, min_sample_size=5000,
             'data_stats_profile':
                 ColumnStatsProfileCompiler(clean_sampled_df, self.options)}
 
-        # use the data labeler by default
-        use_data_labeler = True
-        if options and isinstance(options, StructuredOptions):
-            use_data_labeler = options.data_labeler.is_enabled
-
-        if use_data_labeler:
```

Comment:
IMO, the options should still dictate the usage of the data_labeler whether someone passes it in or not.

That way we have one source of control. Having multiple ways to enable/disable could be confusing.